compiler/rustc_query_system/src/query/config.rs

```diff
@@ -55,9 +55,10 @@ pub trait QueryConfig<Qcx: QueryContext>: Copy {
     fn loadable_from_disk(self, qcx: Qcx, key: &Self::Key, idx: SerializedDepNodeIndex) -> bool;
 
     /// Synthesize an error value to let compilation continue after a cycle.
-    fn value_from_cycle_error(
+    fn execute_fallback(
```

Comment:
Same naming issue, not clear what falls back from what.

Reply:
Could you add some comment explaining what the fallback means, and that currently the query-system-wide failure is a cycle error, but later it can be something else (what?).

And in https://github.com/rust-lang/rust/pull/149319#discussion_r2565175054 as well.